engine/src/main/java/io/camunda/zeebe/engine/processing/bpmn/behavior/BpmnJobBehavior.java

```diff
@@ -55,6 +61,7 @@ public void createNewJob(
 
     final var jobKey = keyGenerator.nextKey();
     stateWriter.appendFollowUpEvent(jobKey, JobIntent.CREATED, jobRecord);
+    jobMetrics.jobCreated(jobType);
```

Comment:
Do we need a metric for job creation? We have another metrics named `element_instance_events_total` in `ProcessEngineMetrics`. If I remember correctly, we can filter it by "type = SERVICE_TASK" and "action=activated". I was wondering if that is enough for job creation metrics. Service task activation corresponds to job creation, isn't it?

Reply:
The issue is then that we need to adjust the query, which I thing is not that easy if you want to have it in one table.

![query](https://user-images.githubusercontent.com/2758593/133319910-83b1b4a4-db91-49b4-9d62-db1539698c00.png)

Reply:
Ok. There might be some way to merge multiple queries, right? But let's keep it simple and have explicit job created metrics. :+1: